### ISSUE TITLE
accessibility check reports empty sidebar container

### ISSUE THREAD

**Issue by ntap-bmegan** [NetAppDocs/cloud-manager-cloud-volumes-ontap#38]
Page: [Cloud Volumes ONTAP for Google Cloud](https://docs.netapp.com/us-en/cloud-manager-cloud-volumes-ontap/google-cloud.html)

This page lacks a sidebar. Is this intentional or an oversight? It's being reported by the accessibility checker. If there is an architectural case for page without sidebar, then we'll need to open a development issue as well. (There is actually an empty sidebar container, which is an accessibility problem.

**Comment by netapp-bcammett** [comment]
This is by design to meet a business need and is temporary.
